Azure/communication-ui-library-ios
# [Bugfix] Updated the participant list UIs to match the latest UX requirement and Android for consistency

## Purpose
Updated the participant list UIs to match the latest UX requirement and Android for consistency
* Muted icon: use the unfilled style as indicated in the spec and on Android
* Updated display name local participant postfix from "(You)" to "(me)"
* Updated participant avatar style conditionally based on unnamed participant
* Updated text colour of unnamed participant name

![Simulator Screen Shot - iPhone 12 Pro - 2022-01-27 at 00 19 19](https://user-images.githubusercontent.com/90668345/151322402-49bdda4f-88cb-4d60-8a16-5feeff2df57a.png)


## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: UX update
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

* Join a call (using group or Teams) (with name filled, or without name filled)
* Launch participant drawer
* Follow **What to Check** section


## What to Check
Verify that the following are valid
In participant drawer: 
* Check if the muted icon style is the unfilled (regular) style as indicated in the spec and on Android 
* Check the display name local participant postfix is "(me)"
* Check if the participant avatar style (abbreviation) is conditionally based on the unnamed participant
* Check if the text colour of unnamed participant name is (#6E6E6E, light) and (#919191, dark)
* Check if the colour of the unfilled (regular) style **mute** icon is (#ACACAC, light) and (#919191, dark)

## Other Information
